docs/release/support_matrix.md

```diff
@@ -1,7 +1,428 @@
+
+Each DAOS engine needs one high-speed network port for communication in the
+DAOS data plane. DAOS Version 2.4 does not support more than one
+high-speed network port per DAOS engine.
+(It is possible that two DAOS engines on a 2-socket server share a
+single high-speed network port for development and testing purposes,
+but this is not supported in a production environment.)
+It is strongly recommended that all DAOS engines in a DAOS system use the same
+model of high-speed fabric adapter.
+Heterogeneous adapter population across DAOS engines has **not** been tested,
+and running with such configurations may cause unexpected behavior.
+Please refer to "Fabric Support" below for more details.
+
+
+## Hardware platforms supported for DAOS Clients
+
+DAOS Version 2.4 supports the x86\_64 architecture.
+
+DAOS clients have no specific hardware dependencies.
+
+Each DAOS client needs a network port on the same high-speed interconnect
+that the DAOS servers are connected to.
+Multiple high-speed network ports per DAOS client are supported.
+Note that a single task on a DAOS client will always use a single network port,
+but when multiple tasks per client node are used then the DAOS agent will
+distribute the load by allocating different network ports to different tasks.
+
+
+## Operating Systems supported for DAOS Servers
+
+The DAOS software stack is built and supported on
+Linux for the x86\_64 architecture.
+
+DAOS Version 2.4 has been primarily validated
+on [Rocky Linux 8.6](https://docs.rockylinux.org/release_notes/8.6/)
+and [openSUSE Leap 15.4](https://en.opensuse.org/openSUSE:Roadmap).
+The following subsections provide details on the Linux distributions
+which DAOS Version 2.4 supports on DAOS servers.
+
+Note that all DAOS servers in a DAOS server cluster (also called _DAOS system_)
+must run the same Linux distribution. DAOS clients that access a DAOS server
+cluster can run the same or different Linux distributions.
+
+
+### SUSE Linux Enterprise Server 15 and openSUSE Leap 15
+
+DAOS Version 2.4 is supported on SLES 15 SP4 and openSUSE Leap 15.4.
+
+General support for SLES 15 SP3 has ended on 31-Dec-2022.
+DAOS nodes running SLES 15 SP3 or openSUSE 15.3
+have to be updated to 15.4 before updating DAOS to version 2.4.
+
+Links to SLES 15 Release Notes:
+
+* [SLES 15 SP3](https://www.suse.com/releasenotes/x86_64/SUSE-SLES/15-SP3/)
+* [SLES 15 SP4](https://www.suse.com/releasenotes/x86_64/SUSE-SLES/15-SP4/)
+
+Links to openSUSE Leap 15 Release Notes:
+
+* [openSUSE Leap 15.3](https://doc.opensuse.org/release-notes/x86_64/openSUSE/Leap/15.3/)
+* [openSUSE Leap 15.4](https://doc.opensuse.org/release-notes/x86_64/openSUSE/Leap/15.4/)
+
+Refer to the [SLES Life Cycle](https://www.suse.com/lifecycle/)
+description on the SUSE support website for information on SLES support phases.
+
+
+### Enterprise Linux 8 (EL8): RHEL 8, Rocky Linux 8, AlmaLinux 8
+
+DAOS Version 2.4 is supported on EL 8.8, and on EL 8.6 with Extended Update Support (EUS).
```

Comment:
We have not actually validated 8.8.  We still run on 8.6 in CI.  I tried bumping to 8.7 but ran into failures that were unreproducible.  This has gone through 2 iterations but I have yet to find time to validate the second iteration.

Reply:
as 8.8 is the current EL8 level, we have to support it (in the sense that we accept tickets against it).  
it's highlighted elsewhere in the doc that most validation was done on 8.6.  
And 8.7 is not relevant anymore as it's EOL.

Reply:
I don't see any tickets in SRS for support of EL 8.7 or 8.8.  Given past experience we do not move distro versions until there is a ticket requesting it as we are not aware of what our customer support requirement/constraints are.

I.e. if we move to 8.8 without a request for it -- without understanding the customer support requirements, we risk building on a point release that will not support customer environments that are still running older versions, such as 8.6 for example.

Reply:
good point, we'll add 2.4 SRS tickets to track EL8 OS validation beyond 8.6

Reply:
reworded doc as "Validation of DAOS Version 2.4 on EL 8.8 is in progress."